spec/services/participants/query_spec.rb

```diff
@@ -222,8 +238,8 @@
         participant3.applications.first.update!(accepted_at: 2.days.ago)
       end
 
-      it "orders applications by accepted_at in ascending order" do
-        expect(query.participants).to eq([participant2, participant3, participant1])
+      it "by default orders by created_at in ascending order" do
```

Comment:
could we update the before/setup to update the created at instead as the setup is now confusing?